extensions/sql-database-projects/package.json

```diff
@@ -309,8 +309,7 @@
           "command": "sqlDatabaseProjects.createProjectFromDatabase"
         },
         {
-          "command": "sqlDatabaseProjects.updateProjectFromDatabase",
-          "when": "false"
+          "command": "sqlDatabaseProjects.updateProjectFromDatabase"
```

Comment:
Why this change?  Did we change our mind and now want to let people launch SC from the command palette?

Reply:
Talked offline; decision from PM